Bottom: edec49e9c93b53129f888dd43dcc18ccb0a79778
Top:    a168f8a03748927cd70d01bafff8e465be163933
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 16:34:46 +0000

Refresh of play-with-quad-edge-data-STEP

---

diff --git a/src/hid/common/step_id.h b/src/hid/common/step_id.h
new file mode 100644
index 0000000..86a4974
--- /dev/null
+++ b/src/hid/common/step_id.h
@@ -0,0 +1,2 @@
+typedef int step_id;
+typedef GList* step_id_list;
